docs/examples/query_engine/multi_doc_auto_retrieval/multi_doc_auto_retrieval.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        api_key="RR3SptbaO2l5Xqb2GbEZtUKXOVRcrDEYhAHw"
</pre><p>could remove this one </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9631/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
lol oops thanks haotian